index.html

```diff
@@ -4,14 +4,162 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>AOD</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Roboto+Slab:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/reset.css" type="text/css">
     <!-- Layout -->
     <link rel="stylesheet" href="css/simple-css-layout.css" type="text/css">
     <link rel="stylesheet" href="css/styles.css" type="text/css">
 </head>
-<body>
-    
+<header class="container-fluid">
+    <div class="nav row">
+        <div class="col-mobile-100">
+            <img class=" col-tablet-hide  col-mobile-land-hide  col-large-tablet-hide col-desktop-hide"src="image/logo.png" alt="Logo AOD">
+            <ul class="nav-list col-mobile-land-33 col-tablet-40 col-desktop-33">
+                <li class="nav-item col-tablet-50">
+                    <a href="#" class="nav-link">about us</a>
+                </li>
+                <li class="nav-item col-tablet-50">
+                    <a href="#" class="nav-link">work</a>
+                </li>
+            </ul>
+            <div class="col-mobile-land-33 col-tablet-20 col-desktop-33">
+                <img class="col-mobile-hide" src="image/logo.png" alt="Logo AOD">
+            </div>
+            <ul class="nav-list col-mobile-land-33 col-tablet-40 col-desktop-33">
+                <li class="nav-item col-tablet-50">
+                    <a href="#" class="nav-link">shop</a>
+                </li>
+                <li class="nav-item col-tablet-50">
+                    <a href="#" class="nav-link">contact</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+    <div class="header-bottom row">
+        <div class="col-mobile-100">
+            <img class="col-large-tablet-hide col-desktop-hide" src="image/mobile-header.jpg" alt="">
+        </div>
+        <div class="col-mobile-hide col-desktop-100">
+            <img src="image/desktop-header.jpg" alt="">
```

Comment:
Tutaj zostawiles pusty atrybut alt co jest dobra praktyka gdy grafika jest czysto dekoracyjna :). To jest szablon strony, wiec dopiero po wprowadzeniu tresci programista lub SEO manager zdecyduje o atrybucie alt :)